controller/internal/routes/authenticate_router.go

```diff
@@ -102,3 +105,18 @@ func (ro *AuthRouter) authHandler(ae *env.AppEnv, rc *response.RequestContext) {
 
 	rc.RequestResponder.RespondWithOk(currentSession, nil)
 }
+
+func (ro *AuthRouter) mapConfigTypeNamesToIds(ae *env.AppEnv, values []string) []string {
+	var result []string
+	if stringz.Contains(values, "all") {
+		return []string{"all"}
+	}
+	for _, val := range values {
+		if configType, err := ae.GetHandlers().ConfigType.Read(val); err == nil && configType != nil {
+			result = append(result, val)
+		} else if configType, err := ae.GetHandlers().ConfigType.ReadByName(val); err == nil && configType != nil {
+			result = append(result, configType.Id)
+		}
```

Comment:
should a fall through branch be added 'just in case'?

Reply:
Could add a debug log statement